api/src/constants.ts

```diff
@@ -1,4 +1,4 @@
-import { DirectionInBoard, Board } from "./types";
+import { DirectionInBoard, Board } from './types';
 
```

Comment:
You could mutualised constants for client and api. Like adding a common package. Keep it in mind for after the demo ;)